### ISSUE TITLE
resizing an iframe

### ISSUE THREAD

**Issue by FutureAndAHope** [nyroDev/nyroModal#54]
Hi,

I am using the following command to open an iframe: 

$.nmManual('images_edit.php', {sizes:{initW:200, initH:1000, w:200, h:1000}});

I have tried various width and heights but no matter what I enter as parameters I can not get the iframe to resize, it opens fine but just does not resize. I have tried using the resize function, after the iframe has loaded but I can't get that to work either. Another thing I tried was to use the class selectors to resize the iframe it works using a time out, but not the built in callback events. If there is no way to resize an iframe what callback should I be using or can i create a custom callback. I guess I could put a function on the page that is being loads that calls parent.do_my_dogy_class_selection_resize() but I would prefer to find a solution that uses existing code.

As a note I had to modify a core file to get the nmManual function to open in an iframe, the following is the change I made (just added Target: '_blank'):

nmManual: function(url, opts) {
$('<a />', {
     href: url, target: '_blank'
     }).nyroModal(opts).trigger('nyroModal');
}

**Comment by Freez** [comment]
I have the same problem, please fix. Im rolling back to version 1.6

**Comment by nyroDev** [comment]
The default iframe size is defined in CSS with:
.nyroModalCont iframe {
  width: 600px;
  height: 400px;
}

Update these rules as you want and it will works great.

**Comment by ashv-zz** [comment]
I'm also using nmManual to open an iframe. The size of the iframe wont always be the same though, so if I set the width and height in the css, the modal window will be the same size for all iframes. Is there a way to set the size using commands?

I've tried to delete the css width and height and then I did exactly what FutureAndAHope did (setting the size using the nmManual options parameters), but had no luck in getting it to work.
